origin.spec

```diff
@@ -82,11 +82,14 @@ install -m 0644 -t %{buildroot}%{_sysconfdir}/sysconfig %{SOURCE10}
 
 mkdir -p %{buildroot}/var/log/%{name}
 
+ln -s %{_bindir}/openshift %{_bindir}/osc
```

Comment:
RPM build fails, this should fix it

-ln -s %{_bindir}/openshift %{_bindir}/osc
+ln -s %{_bindir}/openshift %{buildroot}%{_bindir}/osc

There may be some nuances related to installroot that i may not be thinking of however.